superset-frontend/src/dashboard/util/getPersistentRefreshFrequency.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+export default function getPersistentRefreshFrequency({
+  currentRefreshFrequency = 0,
+  isPersistent = false,
+  dashboardInfo = {},
+}): number {
+  // get original persistent refresh frequency
+  const { metadata }: any = dashboardInfo;
```

Comment:
We know dashboardInfo is an object at least, right? So maybe we can be more specific than `any` type here?

Reply:
how to set object type in typescript?

Reply:
I think using `Record<string, any>` is preferred. you could also make this more specific to something like `{ metadata: { refresh_frequency: number } }`

Reply:
this file is removed.